core/src/main/java/org/apache/carbondata/core/metadata/schema/SchemaReader.java

```diff
@@ -58,6 +58,48 @@ public static CarbonTable readCarbonTableFromStore(AbsoluteTableIdentifier ident
     }
   }
 
+  /**
+   * Read specified schema file as CarbonTable
+   * @param schemaFilePath schema file path
+   * @param conf hadoop configuration
+   * @return CarbonTable object
+   * @throws IOException if IO error occurs
+   */
+  public static CarbonTable readCarbonTableFromSchema(String schemaFilePath, Configuration conf)
+      throws IOException {
+    TableInfo tableInfo = readTableInfoFromSchema(schemaFilePath, conf);
+    CarbonMetadata.getInstance().loadTableMetadata(tableInfo);
+    return CarbonMetadata.getInstance().getCarbonTable("dummy_dummy");
+  }
+
+  /**
+   * Read specified schema file as TableInfo
+   * @param schemaFilePath schema file path
+   * @param conf hadoop configuration
+   * @return TableInfo object
+   * @throws IOException if IO error occurs
+   */
+  public static TableInfo readTableInfoFromSchema(String schemaFilePath, Configuration conf)
+      throws IOException {
+    if (FileFactory.isFileExist(schemaFilePath, FileFactory.FileType.LOCAL) ||
```

Comment:
Use `FileFactory.isFileExist(schemaFilePath)` instead of multiple calls

Reply:
resolved